### DISCUSSION TITLE
@types/css-tree outdated

### DISCUSSION THREAD

**Discussion by scripthunter7** [DefinitelyTyped/DefinitelyTyped#62536] [Issues with a @types package]
The type [definitions of the css-tree package](https://www.npmjs.com/package/@types/css-tree) are outdated, e.g. in List, [isEmpty is a getter, not a method](https://github.com/csstree/csstree/issues/88#issuecomment-1007767433), etc.

CSS-Tree changelog:
https://github.com/csstree/csstree/blob/master/CHANGELOG.md

@erik-kallen @pyoor I saw that you created the type definition, if you are still interested, could you take a look at it please?

[a comment by an automated account is not shown]

**Comment by erik-kallen** [comment]
Hi, @scripthunter7 

Unfortunately, I don't think I will do this, but if you make the changes I will approve them.
